### DISCUSSION TITLE
Aiuda

### DISCUSSION THREAD

**Discussion by BernardoCameron** [Launch-X-Latam/MisionFrontEnd#439] [Team Posts]
Hola chicos, les comento... Quizá algunos están igual que yo en este momento, resulta que vengo recién llegando, acabo de ingresar tanto a Teams como al grupo de GitHub y estoy un poco perdido, alguien podría orientarme para saber mas o menos donde van, me vi los videos de estructura y html de Teams pero no sé qué más hacer. 🥴

Se los agradecería un montón.

**Comment by Lvelmo** [comment]
Yo ando igual hasta hoy me pude vincular

**Comment by gumodi35** [comment]
Que tal gente, pues primero relax... El tema del ingreso así es, tarda un poco. 🕐 <br>
En resumen las sesiones están grabas 🎥 y las pueden ver ondemand.<br>
Si apenas les dieron el acceso al Teams, tienen que esperar que les habiliten 🧑‍🏭 el grupo.<br>
Lo que se ha visto son temas de toma de requerimientos, ciclo de vida del software 🖥️  y hoy un vistazo de HTML. <br>
En otros hilos 🔗 pasaron el link de las seciones grabadas, pero igual van a poder tomarlas cuando tengan tiempo.<br>
Si tienen duda de algo, pues aquí está el foro. Saludos! 👍

**Comment by BernardoCameron** [comment]
Aun no hay que hacer ni una práctica con la que deba ponerme al día?

**Comment by gumodi35** [comment]
El repo de front están las prácticas, de la toma de requerimientos, del buyer persona, del abogabot, diseño del mock-up de la aplicación usando Figma. Y hoy solo lo del HTML. Pero relax... Hay tiempo, a tu ritmo, no hay prisas.

**Comment by TommyRDIce** [comment]
master ando 

> gumodi35 19 hours ago

master estoy igual de perdido Ya que para acceder al grupo de Teams tengo que realizar el Onboarding y el link que me llego al correo para realizarlo no funciona, no quiero atesarme más porque recién empiezo y quiero comenzar a tomas las clases ya


si me podría orientar donde realizar el Onboarding  para luego poder entrar al grupo de Teams y ver las grabaciones seria de muchísima ayuda

**Comment by gumodi35** [comment]
Que tal Tommy! :+1: 
En el repo de la **organizacion** vas a encontrar los repositorios, si aun no estas en el grupo de Teams dirigete a ese repo **Onboarding** y sigue las intrucciones. <br>
![onboarding-1](https://user-images.githubusercontent.com/80011796/194729629-3e06586a-303c-4f2b-a979-5422077b5fd2.png)
<br>
Realiza todas las instrucciones que dice el repositorio  **Onboarding** y cuando hayas avanzado sigue al apartado de acceder a Teams <br>
![onboarding-2](https://user-images.githubusercontent.com/80011796/194729756-129ca028-5df2-468c-828a-0e29b48ca735.png)
<br>
Eso te redirigira a un formulario llenalo con tu informacion y eso te genera el acceso al grupo de Teams.<br>
Eso deberia ser instantaneo, cuando ingresas por primera vez quiza no veas el grupo de Mision Frontend, eso es normal, espera a que te de acceso a veces es despues de un rato, a veces al dia siguiente. <br>
Ya en el grupo vas a ver las seciones que se han avanzdo. <br>
Toda la informacion esta en el repositorio **Mision Frontend** la clase del viernes fue HTML. <br>
![Onboarding-3](https://user-images.githubusercontent.com/80011796/194729898-5ca21554-9907-4e35-b0b2-a9554002b7b3.png) <br>
No te estreses, tienes tiempo para realizar las actividades y de ponerte al dia, lo importante es que vayas aprendiendo. Checa esto y cualquier duda manda DM. Saludos! :100:

**Comment by AldairNava** [comment]
Buenas disculpen, dónde puedo encontrar el formulario para dar el link de mi repositorio?

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
